src/index.html

```diff
@@ -11,6 +11,89 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+
+      action="/"
+
+      class="first-form  all-forms"
+
+      method="get"
+
+      data-qa="big"
+
+    >
```

Comment:
delete redundant empty lines
```suggestion
    <form
      action="/"
      class="first-form  all-forms"
      method="get"
      data-qa="big"
    >
```

Reply:
check it through all file